rotkehlchen/tests/unit/test_ens.py

```diff
@@ -88,20 +89,42 @@ def test_ens_reverse_lookup(ethereum_manager):
     """This test could be flaky because it assumes
         that all used ens names exist
     """
-    reversed_addr_0 = to_checksum_address('0x71C7656EC7ab88b098defB751B7401B5f6d8976F')
-    reversed_addr_1 = ethereum_manager.ens_lookup('lefteris.eth')
-    expected = {reversed_addr_0: None, reversed_addr_1: 'lefteris.eth'}
-    assert ethereum_manager.ens_reverse_lookup([reversed_addr_0, reversed_addr_1]) == expected
-
-    reversed_addr_2 = ethereum_manager.ens_lookup('abc.eth')
-    reversed_addr_3 = ethereum_manager.ens_lookup('rotki.eth')
-    # reversed_addr_4 has not configured ens name resolution properly
-    reversed_addr_4 = string_to_ethereum_address('0x5b2Ed2eF8F480cC165A600aC451D9D9Ebf521e94')
-    expected = {reversed_addr_2: 'abc.eth', reversed_addr_3: 'rotki.eth', reversed_addr_4: None}
-    queried_ens_names = ethereum_manager.ens_reverse_lookup(
-        [reversed_addr_2, reversed_addr_3, reversed_addr_4],
+    contract_calls = 0
+
+    def mock_call_contract(contract, ethereum, **kwargs):
```

Comment:
you don't need to do all this.

You can just mock it by making mock same as the actual call.

And then check the `call_count` property of the mock. We do this elsewhere in the code too.

Reply:
Nice, didn't know it, thanks